### PR TITLE
Task #4 Method checkValue() in class Game

### PR DESCRIPTION
**Pull request by anorberciak**

### CODE REVIEW

File: tests/GameTests.cpp [hunk 1]
```diff
@@ -24,3 +24,27 @@ TEST_F(GameTests, initial_score_shoud_be_zero)
...
+    ASSERT_EQ(10, game.checkValue("2/"));
+}
+
+TEST_F(GameTests, score_should_be_nine)
+{
+    //WHEN
+    game.checkValue("45");
+    //THEN
+    ASSERT_EQ(9, game.checkValue("45"));
+}
```
Comment (Angan7a) [inline tests/GameTests.cpp]: Could you add 2 more tests, please - when is miss. 
ASSERT_EQ(7, game.checkValue("-7"));
ASSERT_EQ(3, game.checkValue("3-"));

**Comment by Angan7a** [comment]
@anorberciak nice job, but could you rename this pull request to "Task #10 Method checkValue() in class Game", please. Could you made little changed in code, please.

File: src/Game.cpp [hunk 2]
```diff
@@ -15,3 +15,18 @@ int Game::score() const
...
+int Game::checkValue(const std::string & value)
+{
+    int value_;
+    if ((value[0] == 'X') || (value[1] == '/'))
+    {
+        value_ = 10;
+    }
+    else
+    {
+        value_= ((value[0] - '0')+(value[1] - '0'));
```
Comment (Tomek52) [inline src/Game.cpp]: @anorberciak you can write function asciiToDec, the code will be cleaner

Comment (LordLukin) [reply src/Game.cpp]: https://en.cppreference.com/w/cpp/string/basic_string/stol

File: src/Game.cpp [hunk 3]
```diff
@@ -15,3 +15,18 @@ int Game::score() const
     //needs implementation after made checkValueFrame()
     return 0;
 }
+
+int Game::checkValue(const std::string & value)
+{
+    int value_;
```
[the remark this answers is not in the document]
Comment (anorberciak) [reply src/Game.cpp]: Thank you for being vigilant ;)

File: src/Game.cpp (hunk 2 above)
Comment (anorberciak) [reply src/Game.cpp]: thx @LordLukin and @Tomek52 , I just finished it :)

File: tests/GameTests.cpp (hunk 1 above)
Comment (anorberciak) [reply tests/GameTests.cpp]: Ok

**Comment by Angan7a** [comment]
For now for me is OK.

**Pull request merged**